src/components/MessageList/MessageList.tsx

```diff
@@ -210,6 +215,8 @@ export type MessageListProps<
    * Defaults to and accepts same props as: [TypingIndicator](https://getstream.github.io/stream-chat-react-native/#typingindicator)
    */
   TypingIndicator?: React.ComponentType<TypingIndicatorProps>;
+  NewMessageNotification?: React.ComponentType<MessageNotificationProps>;
```

Comment:
```suggestion
  MessageNotification?: React.ComponentType<MessageNotificationProps>;
```

Reply:
also put this and the onListScroll in alphabetical order like the rest of the types here